src/lib/LavalinkNode.ts

```
@@ -1,264 +1,489 @@
+	 * @param reason - The reason why the WebSocket was closed
+	 */
+	private onClose(code: number, reason: Buffer): void {
+		this._sessionUpdated = false;
+		this.manager.emit("disconnect", code, reason.toString(), this);
+
+		this.ws?.removeAllListeners();
+		this.ws = null;
+
+		switch (code) {
+			case 1000:
+				if (reason.toString() === "destroy") return clearTimeout(this._reconnect);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)